### PR TITLE
APP-6846: Adding the GetBillingConfig CLI endpoint

### PR DESCRIPTION
**Pull request by RoxyFarhad**
The new command from the scope:
```
viam organization billing-service get-config --org-id=<org-id>
```

Manual Testing:
Screenshot 2024-12-06 at 2 16 01 PM

### CODE REVIEW

File: cli/client.go [hunk 1]
```diff
@@ -177,6 +177,44 @@ func (c *viamClient) organizationsSupportEmailGetAction(cCtx *cli.Context, orgID
...
+
+	printf(cCtx.App.Writer, " --- Billing Address --- ")
+	printf(cCtx.App.Writer, "Address Line 1: %s", resp.BillingAddress.GetAddressLine_1())
+	if resp.BillingAddress.GetAddressLine_2() != "" {
+		printf(cCtx.App.Writer, "Address Line 2: %s", resp.BillingAddress.GetAddressLine_2())
+	}
+	printf(cCtx.App.Writer, "City: %s", resp.BillingAddress.GetCity())
+	printf(cCtx.App.Writer, "State: %s", resp.BillingAddress.GetState())
+	printf(cCtx.App.Writer, "Postal Code: %s", resp.BillingAddress.GetZipcode())
+	printf(cCtx.App.Writer, "Country: %s", "USA")
```
Comment (benjirewis) [inline cli/client.go:214]: 😆 no international customers?

**Review by benjirewis — APPROVED** [review]
LGTM; thanks.

File: cli/client_test.go [hunk 2]
```diff
@@ -237,6 +237,47 @@ func TestGetSupportEmailAction(t *testing.T) {
 	test.That(t, out.messages[0], test.ShouldContainSubstring, "test-email")
 }
 
+func TestGetBillingConfigACtion(t *testing.T) {
```
Comment (JosephBorodach) [inline cli/client_test.go:240]: []
```suggestion
func TestGetBillingConfigAction(t *testing.T) {
```

Comment (RoxyFarhad) [reply cli/client_test.go:240]: thank you will fix!

**Review by JosephBorodach — APPROVED** [review]
have we manually tested? If so LGTM :shipit:

File: cli/client.go (hunk 1 above)
Comment (RoxyFarhad) [reply cli/client.go:214]: haha not yet

**Pull request merged**